src/Components/EnvironmentPicker/EnvironmentPickerManager.ts

```diff
@@ -13,10 +13,15 @@ import {
 export const getEnvironmentDisplayText = (env: string | IAzureResource) => {
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186237565-5e5662fc-78d6-4ad1-b0a8-09a2d1acc789.png)

Reply:
this is something we can discuss and implement in a separate PR

Reply:
this is the second PR. You already punted it from the PR where you added this. :)